master-branch-pipeline.yml

```diff
@@ -22,7 +23,7 @@ jobs:
       jdkArchitectureOption: x64
       jdkSourceOption: LocalDirectory
       jdkFile: "$(Build.ArtifactStagingDirectory)/java11.tar.gz"
-      jdkDestinationDirectory: ""
+      jdkDestinationDirectory: "$(Build.ArtifactStagingDirectory)/jdk"
```

Comment:
What is what is this line doing, specifically?  Does this depend on some additional configuration that is happening elsewhere? Maybe in the VM setup? Would be good to document inline here as a comment.

Reply:
This is a gotcha to the jdk unzip process.

Initially, when jdkDestinationDirectory was left blank, it would uncompress to the same directory as the git clone. As part of that process, it would blow away the contents of that directory.

This explicitly puts it in a safe place where it won't overlap with project code.

Reply:
Gotcha, thanks!